runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/RuntimeUtilities.java

```diff
@@ -416,6 +426,19 @@ public static void writeStreamToFile(File file, byte[] source, boolean append) {
 
     }
 
+    public static void writeStringToFile(String filename, String source, boolean append) {
+        try (FileWriter fw = new FileWriter(filename, append)) {
```

Comment:
From the FileWriter docs --> `Whether or not a file is available or may be created depends upon the underlying platform.`

I suppose the file is created if missing for both Windows and Linux

Reply:
good catch, we do not have an FPGA on Windows to test it. But I will try to call the `FileWriter` and see if it is ok.

Reply:
I can confirm that in bot OS, the file is created if it does not exist.